addon/components/geojson-layer.js

```diff
@@ -0,0 +1,61 @@
+import Ember from 'ember';
+import BaseLayer from 'ember-leaflet/components/base-layer';
+import PathLayer from 'ember-leaflet/components/path-layer';
+/* global L */
+
+/**
+ * An ember-leaflet wrapper for L.geoJson, which renders GeoJson data onto a
+ * map as features.
+ *
+ * Takes:
+ *   - geoJSON: the GeoJSON object to render
+ *   - all standard leaflet options for L.geoJson
+ */
+export default BaseLayer.extend({
+  leafletRequiredOptions: ['geoJSON'],
+
+  leafletOptions: [
+    'stroke', 'color', 'weight', 'opacity', 'fill', 'fillColor',
+    'fillOpacity', 'fillRule', 'dashArray', 'lineCap', 'lineJoin',
+    'clickable', 'pointerEvents', 'className', 'pointToLayer',
+    'style', 'onEachFeature', 'filter', 'coordsToLatLng'
+  ],
+
+  leafletEvents: [
+    'click', 'dblclick', 'mousedown', 'mouseover', 'mouseout',
+    'contextmenu', 'add', 'remove', 'popupopen', 'popupclose'
+  ],
+
+  geoJSON: null,
```

Comment:
To conform the other components, you don't need to specify required options. You can remove this line.